README.md

```diff
@@ -75,6 +76,7 @@ with each file. In JSON, the metadata looks something like this:
             "end": "2015-03-20T23:59:59.114Z",
             "where": "webserver02",
             "what": "syslog",
+            "who": "webteam",
```

Comment:
As we discussed I don't think this is appropriate as it dose not describe the data.